app/views/layouts/application.html.erb

```diff
@@ -1,39 +1,39 @@
 <!DOCTYPE html>
 <html>
 <head>
-  <title>PodcastReviewApp</title>
+  <title>PodcastR</title>
   <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.7.0/css/all.css" integrity="sha384-lZN37f5QGtY3VHgisS14W3ExzMWZxybE1SJSEsQp9S+oqd12jhcu+A56Ebc1zFSJ" crossorigin="anonymous">
+  <link href="https://fonts.googleapis.com/css?family=Nunito:600" rel="stylesheet">
   <%= stylesheet_link_tag 'application', media: 'all' %>
-
-    <%= javascript_include_tag "vendor/modernizr" %>
-
+  <%= javascript_include_tag "vendor/modernizr" %>
   <%= csrf_meta_tags %>
 </head>
 <body>
 
   <nav class="top-bar">
     <ul class="title-area">
       <li class="name">
-        <h1><a href="/">PodcastR</a></h1>
+        <h1><a href="/"><i class="fas fa-headphones"></i> PodcastR</a></h1>
       </li>
     </ul>
 
     <section class="top-bar-section">
       <ul class="right">
 
         <%- if current_user -%>
-          <li><%= link_to 'Sign Out', destroy_user_session_path, method: :delete %></li>
+          <li class="devise-nav-bar"><%= link_to 'Sign Out', destroy_user_session_path, method: :delete %></li>
         <%- else -%>
-          <li><%= link_to 'Sign Up', new_user_registration_path %></li>
-          <li><%= link_to 'Sign In', new_user_session_path %></li>
+          <li class="devise-nav-bar"><%= link_to 'Sign Up', new_user_registration_path %></li>
+          <li class="devise-nav-bar"><%= link_to 'Sign In', new_user_session_path %></li>
         <%- end -%>
       </ul>
     </section>
   </nav>
 
 <div>
   <% if current_user && current_user.profile_photo.url%>
-  <img class="prof-pic" src="<%=current_user.profile_photo.url%>">
+  <img class=“prof-pic” src="<%=current_user.profile_photo.url%>">
```

Comment:
Oh no! You've fallen into the "curly quotes" trap. I'm surprised this is working for you -- even if it is, please replace them with straight quotes so I can sleep at night.

(Ask Gabi for a story about her curly-quote debugging adventures -- long story short, curly quotes != straight quotes)